### PR TITLE
Add nagl

### PR DESCRIPTION
**Pull request by SimonBoothroyd**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by SimonBoothroyd** [comment]
@conda-forge/help-python this should be ready to go! The package does not support windows so the CI failure due to missing `pytorch` should not be an issue

File: recipes/nagl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimonBoothroyd/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: cca4b5c975405d720368a337f2908586ef44185934f7ac56c98bb4d078151153
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (dopplershift) [inline recipes/nagl/meta.yaml:15]: Would be good to go ahead and add the entry point for the CLI:
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
  entry_points:
    - nagl = nagl.cli:cli
```

File: recipes/nagl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - openff-utilities
+    - pydantic
+    - click
+    - click-option-group
+    - pytorch
+    - tqdm
+
+test:
+  imports:
+    - nagl
```
Comment (dopplershift) [inline recipes/nagl/meta.yaml:32]: Maybe test the cli too?
```suggestion
  imports:
    - nagl
  commands:
    - nagl
```

File: recipes/nagl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+about:
+  home: https://github.com/SimonBoothroyd/nagl
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (dopplershift) [inline recipes/nagl/meta.yaml:38]: Probably should include the 3rd party stuff too:
```suggestion
  license: MIT AND BSD-3-CLAUSE
  license_file:
    - LICENSE
    - LICENSE-3RD-PARTY
```

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Comment by SimonBoothroyd** [comment]
Thanks @dopplershift for catching these! They should be resolved by 5422805 and 0ccea6e

**Review by dopplershift — APPROVED** [review]
Ignoring failure on Windows due to noarch and missing dependencies.

**Comment by SimonBoothroyd** [comment]
Thanks @dopplershift !

**Pull request merged**